x/staking/keeper/keeper.go

```diff
@@ -84,7 +128,7 @@ type Keeper struct {
 	// LastTotalPower value: LastTotalPower
 	LastTotalPower collections.Item[math.Int]
 	// ValidatorUpdates value: ValidatorUpdates
-	ValidatorUpdates collections.Item[types.ValidatorUpdates]
+	ValidatorUpdates collections.Item[types.ModuleValidatorUpdates]
```

Comment:
does anyone use this value? seems we only set it

Reply:
That's a good point 👍🏾 Let me ask, we don't use it ourself indeed.